api/buffer/design.go

```diff
@@ -377,9 +377,6 @@ var CreateReceiverRequest = Type("CreateReceiverRequest", func() {
 		Description("Optional ID, if not filled in, it will be generated from name. Cannot be changed later.")
 	})
 	receiverFields()
-	Attribute("exports", ArrayOf(CreateExportData), func() {
-		Description("List of exports, max 20 exports per a receiver.")
-	})
 	Required("name")
```

Comment:
Our APIs are not strictly REST.

In some places, it is possible to list or modify sub-entities together with the parent entity, with one API call.

It is a concession to the UI, which can thus load/modify everything with one API call, e.g. GET all sources + sinks.

Here, with a Source, you can create its Sinks as well. 

But it was not used in the UI, there is a wizard with 2 steps. So I delete it, it is unused.